types/rdf-utils-dataset/resourcesToGraph.d.ts

```diff
@@ -14,6 +13,5 @@ interface Options<F extends Factory> {
 
 type DatasetOf<F> = F extends RDF.DatasetCoreFactory<any, any, infer D> ? D : never;
 
-declare function resourcesToGraph<F extends Factory = Factory<QuadExt, QuadExt, DatasetExt>>(_input: DatasetIndexed<RDF.BaseQuad, RDF.BaseQuad>, options?: Options<F>): DatasetOf<F>;
-
-export = resourcesToGraph;
+export default function resourcesToGraph<F extends Factory = Factory<QuadExt, QuadExt, DatasetExt>>(_input: RDF.DatasetCore<RDF.BaseQuad, RDF.BaseQuad>, options?: Options<F>): DatasetOf<F>;
+export {};
```

Comment:
Why not have `DatasetExt` as the generic here?

Reply:
I cannot answer as I'm not the original author of these types